core/test/functional/routes/api/public_api_spec.js

```diff
@@ -161,6 +161,33 @@ describe('Public API', function () {
             });
     });
 
+    it('browse posts with page filter', function (done) {
+        request.get(testUtils.API.getApiQuery('posts/?client_id=ghost-admin&client_secret=not_available&filter=page:true'))
```

Comment:
How many filter tests do we have now? 🤔 
I'd suggest one per admin and one per public API, that's it.

Reply:
We have 9 filter tests atm:
- 2 from posts_spec: cover featured flag
- 7 from public_api_spec: all cover different combinations of filtering by own properties, nested properties, computed ones. They don't repeat and test a bit different contexts, find them rather useful than a waste of cpu cycles :)  

Would definitely not add more of them but ow what we have would rather remove the featured flag ones and leave the rest.

Started writing this comment before the standup. Now that it's 200% clear we want to aim for even less test, I'd leave 2-3 of them that cover the widest cases.